content/docs/06-integrations/00-prometheus-agent.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Prometheus Agent'
+metaTitle: 'Prometheus Agent'
+metaDescription: 'Prometheus Agent Monitoring Pack'
+hiddenFromNav: true
+type: "integration"
+hideToC: false
+category: ['monitoring']
+logoUrl: 'https://registry.spectrocloud.com/v1/prometheus-operator/blobs/sha256:64589616d7f667e5f1d7e3c9a39e32c676e03518a318924e123738693e104ce0?type=image/png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Prometheus Agent
+
+Prometheus is an open-source monitoring and alerting system that is designed to collect and analyze metrics from various systems and services. 
+
+Prometheus is built around a time-series database that stores metrics data. It uses a flexible querying language called PromQL to extract and process metrics data. Prometheus also has a powerful alerting system that can be used to send notifications when specific conditions are met.
+
+Prometheus can be used to monitor a wide range of systems and services, including servers, containers, databases, and applications. It can be deployed in a variety of environments, including on-premises, cloud, and hybrid setups.
+
+The Prometheus Agent pack works in tandem with the [Prometheus Operator pack](/integrations/prometheus-operator). Check out the guides [Deploy Monitoring Stack](/clusters/cluster-management/monitoring/deploy-monitor-stack) and [Enable Monitoring on Host Cluster](/clusters/cluster-management/monitoring/deploy-agent) to learn how to create a monitoring stack with Prometheus for your Palette environment.
+
+
+# Versions Supported
+
+**19.0.X**
+
+# Prerequisites
+
+* A host cluster that has the [Prometheus Operator pack](/integrations/prometheus-operator) installed.
+
+# Parameters
+
+The Prometheus agent supports all the parameters exposed by the Prometheus Helm Chart. Refer to the [Prometheus Helm Chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus#configuration) documentation for details.
+
+From a Palette perspective, there is one required parameter you must provide a value:
+
+<br />
+
+```yaml
+charts:
+  prometheus:
+    server:
+      remoteWrite:
+        - url: ""  
+```
+
+The `remoteWrite.url` is exposed by the [Prometheus Operator pack](/integrations/prometheus-operator) when installed in a cluster. The Prometheus server URL can be found by reviewing the details of the host cluster hosting the Prometheus server. Use the URL exposed by the Prometheus service. 
+
+The following image displays a host cluster with the Prometheus Operator pack installed. Use the URL exposed for port 9090 to populate the `remoteWrite.url` parameter.
+
+![A view of the cluster details page with a highlighted box around the Prometheus service URL](/integrations_prometheus-agent_cluster-detail-view.png)
+
+<br />
+
+<WarningBox>
+
+The Prometheus server URL must be in the format of `http://HOST:PORT/api/v1/write`. 
```

Comment:
I changed this slightly elsewhere. If you think it's unnecessary, the call is yours.